outline.md

```diff
@@ -0,0 +1,59 @@
+## Growing an Inner Source Program
+
+45 min
+
+## Short Description
+
+"Inner source is a great idea! Go do inner source!"
+You're the dedicated champion for inner source in your company.
+What do you do now?
+How do you realistically affect the behavior of dozens or hundreds of teams to the point where robust and pervasive inner sourcing is a normal part of the way that engineering is done?
+
+This presentation shares principles, practical anecdotes, and relatable examples on this situation gleaned from experience at Nike.
+While not claiming all of the answers, it is insightful summary of over a year of work in the area.
+
+## Audience
+This presentation will be given at [the ISC Spring Summit](http://paypal.github.io/InnerSourceCommons/events/isc-spring-2018-agenda/).
+It is intended for those that are evangelizing inner source in a specific company.
+
+## Detailed description
+
+#### Outline
+
+An inner source program is the set of tools, documentation, training, and so forth provided so that any team can participate in inner source effectively as either a maintainer or contributor.
+To frame the talk, we'll explain the goals of an inner source program as well as the context within-which we've begun to pursue those goals.
+Even though the learning in this presentation is occurring in that context, we suspect that much of what's here applies generally.
+
+When building an inner source program you should have a vision of what services you will provide to facilitate inner sourcing _as well as_ a strategy you will employ to roll out and make effective these services in your organization.
+We'll explain principles, resources, and specific examples that can aid and guide in doing so.
+
+The services of the inner source program should provide two things.
+First, they must teach, define, and document the type of culture that is conducive to inner source.
+Second, there must be software that reinforces that culture while simultaneously lowering the barrier to entry to participating in that culture.
+Examples of these services include:
+
+* Documentation on how to do inner source (and relation of that documentation to [InnerSourcePatterns](http://innersourcecommons.org/patterns)).
+* Software for automated, two-way discovery of inner source projects (maintainer to contributor/user and contributor/user to maintainer).
```

Comment:
Scout seems like a one way discovery mechanism and its a bit clunky to discover InnerSource projects.  I'm not sure we can lean on this bullet yet.  Are we describing the ideal state or the state where we are today?

Reply:
ideal state.  The point (in my mind) of the presentation is to present a case for working this way so that as we do it in the upcoming months others may be interested in working with us as we go about implementing these ideas.